File: main/contrib/Windows API Code Pack 1.1/source/Samples/Shell/CommonFileDialogsDemo/CS/CommonFileDialogsDemo/SubItemsForm.cs
// Copyright (c) Microsoft Corporation.  All rights reserved.

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace Microsoft.WindowsAPICodePack.Samples.ShellObjectCFDBrowser
{
    public partial class SubItemsForm : Form
    {
        private int itemsCount = 0;
        public SubItemsForm()
        {
            InitializeComponent();

            listView1.LargeImageList = imageList1;
        }

        public void AddItem(string name, Image image)
        {
            imageList1.Images.Add(image);
            listView1.Items.Add(new ListViewItem(name, itemsCount));
            itemsCount++;
        }
    }
}

File: main/contrib/Windows API Code Pack 1.1/source/Samples/Shell/CommonFileDialogsDemo/VB/CommonFileDialogsDemo/SubItemsForm.vb
' Copyright (c) Microsoft Corporation.  All rights reserved.


Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace Microsoft.WindowsAPICodePack.Samples.ShellObjectCFDBrowser
	Partial Public Class SubItemsForm
		Inherits Form
		Private itemsCount As Integer = 0
		Public Sub New()
			InitializeComponent()

			listView1.LargeImageList = imageList1
		End Sub

		Public Sub AddItem(ByVal name As String, ByVal image As Image)
			imageList1.Images.Add(image)
			listView1.Items.Add(New ListViewItem(name, itemsCount))
			itemsCount += 1
		End Sub
	End Class
End Namespace
